api/METRICS.md

```diff
@@ -86,21 +86,186 @@ group_max_from_summary(summary)
 
 |metric|variants|task|notes|aif360|
 |------|--------|-----|----|------|
-|`selection_rate`| G,D,R,Min | class | | &#x2713; |
-|`demographic_parity`| D,R | class | `selection_rate_difference`, `selection_rate_ratio` | `statistical_parity_difference`, `disparate_impact`|
-|`accuracy_score`| G,D,R,Min | class | sklearn | `accuracy` |
-|`balanced_accuracy_score` | G | class | sklearn | - |
-|`mean_absolute_error` | G,D,R,Max | class, reg | sklearn | class only: `error_rate` |
 |`confusion_matrix` | G | class | sklearn | `binary_confusion_matrix` |
 |`false_positive_rate` | G,D,R | class | | &#x2713; |
 |`false_negative_rate` | G | class | | &#x2713; |
 |`true_positive_rate` | G,D,R | class | | &#x2713; |
 |`true_negative_rate` | G | class | | &#x2713; |
-|`equalized_odds` | D,R | class | max of difference or ratio under `true_positive_rate`, `false_positive_rate` | - |
-|`precision_score`| G | class | sklearn | &#x2713; |
-|`recall_score`| G | class | sklearn | &#x2713; |
-|`f1_score`| G | class | sklearn | - |
-|`roc_auc_score`| G | prob | sklearn | - |
-|`log_loss`| G | prob | sklearn | - |
-|`mean_squared_error`| G | prob, reg | sklearn | - |
-|`r2_score`| G | reg | sklearn | - |
+|`selection_rate`| G,D,R | class | | &#x2713; |
+|`demographic_parity`| D,R | class | `selection_rate_difference`, `selection_rate_ratio` | `statistical_parity_difference`, `disparate_impact`|
+|`equalized_odds` | D,R | class | max of difference or min ratio under `true_positive_rate`, `false_positive_rate` | - |
+|`accuracy_score`| G,D,R,Min | class | sklearn | `accuracy` |
+|`balanced_accuracy_score` | G,Min | class | sklearn | - |
+|`precision_score`| G,Min | class | sklearn | &#x2713; |
+|`recall_score`| G,Min | class | sklearn | &#x2713; |
+|`f1_score`| G,Min | class | sklearn | - |
+|`roc_auc_score`| G,Min | prob | sklearn | - |
+|`log_loss`| G,Max | prob | sklearn | - |
+|`mean_prediction`| G | prob, reg | | - |
+|`mean_absolute_error` | G,D,R,Max | class, reg | sklearn | class only: `error_rate` |
+|`mean_squared_error`| G,Max | prob, reg | sklearn | - |
+|`r2_score`| G,Min | reg | sklearn | - |
+|`_mean_overprediction` | G | class, reg | | - |

```

Comment:
When do you think you'll have an alternative to these underscores?

Reply:
I'd like to ask the community what their thoughts are given that we're doing somewhat non-standard things here.